### ISSUE TITLE
Breakpoints triggered but continuation commands (run, step, stop) don't work

### ISSUE THREAD

**Issue by hdodov** [martomo/SublimeTextXdebug#195]
If I add a breakpoint in a file and open it in the browser, its execution correctly halts and the Xdebug panels in Sublime are populated accordingly. However, the continuation functionality doesn't work:

- The yellow triangle in the gutter showing where the script halted doesn't appear.
- The continuation commands in Sublime's command palette do not appear.
- If I right click wherever, the commands do not show up in the Xdebug dropdown.
- I even tried to use the hotkeys (Ctrl+Shift+F5, and the rest) in hopes of making it work, but nothing happens.

If I refresh the page, it's not halted and executes normally, while the Xdebug panels are left unchanged. That's probably because the debugger is stuck in the previous execution which never finished. Basically the only way to move on is to close the debugging session, open a new one, and refresh the page, which brings me to another halted execution that can't be continued.

### Sublime Text
__Operating system:__
Windows 10.0.17134 Build 17134
__Installed version/build:__
3.2, Build 3200
__Python version:__
3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)] on win32

### Server
__Operating system:__
Windows (localhost)
__PHP/Xdebug version:__
PHP 7.2.7 (cli) (built: Jun 19 2018 23:43:47) ( ZTS MSVC15 (Visual C++ 2017) x86 )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Xdebug v2.7.0, Copyright (c) 2002-2019, by Derick Rethans

## Configuration

__php.ini/xdebug.ini__
```ini
[Xdebug]
zend_extension=C:\xampp\php\ext\php_xdebug-2.7.0-7.2-vc15.dll
xdebug.remote_log="C:\Windows\Temp\Xdebug\remote.log"
xdebug.remote_enable=1
xdebug.remote_host="127.0.0.1"
xdebug.remote_port=9000
xdebug.remote_handler="dbgp"
xdebug.remote_mode=req
xdebug.remote_connect_back=0
```

__Packages/User/Xdebug.sublime-settings__
```js
{
  "url": "http://localhost/ikarov",
  "host": "127.0.0.1",
  "path_mapping": {
    "/ikarov" : "C:\\xampp\\htdocs\\ikarov"
  },
  "super_globals": false, // fixes empty panels
  "close_on_stop": true,
  "debug": true
}
```

## Logs

__Packages/User/Xdebug.log:__
[Xdebug.log](https://github.com/martomo/SublimeTextXdebug/files/2991946/Xdebug.log)

**Comment by Daniel-Yonkov** [comment]
@hdodov - See #194 - I have the same issue and it's due to version of Xdebug and PHP - seems like currently supported version is 7.1 (PHP) and 2.5.5 (XDebug)

**Comment by martomo** [comment]
Closing this issue as this should be resolved with release 1.1.2, in case the issue was not resolved please let me know.
